nimble/host/audio/services/bass/src/ble_audio_svc_bass.c

```diff
@@ -477,8 +479,7 @@ ble_svc_audio_bass_modify_source(uint8_t *data, uint16_t data_len, uint16_t conn
 
     operation.modify_source.pa_interval = get_le16(&data[offset]);
     offset += 2;
-    operation.modify_source.num_subgroups = get_le16(&data[offset]);
-    offset += 2;
+    operation.modify_source.num_subgroups = get_le16(&data[offset++]);
```

Comment:
is `offset++`  intended here?

Reply:
It's moved below - this was a mistake. We have to move offset after reading number of subroups (which size is 1 byte - but I'm sure u know that :) ) 
![image](https://github.com/user-attachments/assets/8d4c2a05-bdf6-4097-a4bd-aaa4874f0349)